exercises/clock/.meta/generator/clock_case.rb

```diff
@@ -9,27 +9,30 @@ def name
   end
 
   def workload
-    property == 'equal' ? compare_clocks : simple_test
+    case property
+    when 'add' then arithmetic_test('+', add)
+    when 'create' then create_test
+    when 'equal' then compare_clocks
+    when 'subtract' then arithmetic_test('-', subtract)
+    end
   end
 
   private
 
+  ASSERTION_STR = 'assert_equal %s'
+  CLOCK_STR = 'Clock.at(%d, %d)'
+
   def compare_clocks
     "clock1 = Clock.at(#{clock1['hour']}, #{clock1['minute']})
     clock2 = Clock.at(#{clock2['hour']}, #{clock2['minute']})
     #{assert} clock1 == clock2"
   end
 
-  def simple_test
-    [
-      "assert_equal #{expected.inspect}, ",
-      "#{'(' if add_to_clock}Clock.at(#{hour}, ",
-      "#{minute})#{add_to_clock}#{')' if add_to_clock}.to_s"
-    ].join
+  def create_test
+    "#{ASSERTION_STR}, #{CLOCK_STR}.to_s" % [expected.inspect, hour, minute]
```

Comment:
I like the idea here, simplifying test construction and removing the conditionals, but don't like this specific change.
It squashes the assertion, the expected value, and the subject of the test into one without any clarity about which is which. 

What about something like:
```ruby 
actual = "Clock.at( #{hour}, #{minute} )"
"assert_equal #{expected.inspect}, #{actual}"
```